src/mon/PGMap.cc

```diff
@@ -2057,9 +2057,9 @@ void PGMap::dump_fs_stats(
         << stringify(si_t(osd_sum.kb_used*1024));
     float used = 0.0;
     if (osd_sum.kb > 0) {
-      used = ((float)osd_sum.kb_used / osd_sum.kb);
+      used = ((float)(osd_sum.kb - osd_sum.kb_avail) / osd_sum.kb);
```

Comment:
@sage both changes (here and the one OSD) are from you, maybe you could help double check it?